JDBCEmailRepository.java

```diff
@@ -0,0 +1,142 @@
+package com.example.demo.repository;
+
+import com.example.demo.Email;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+@Repository
+public class JDBCEmailRepository implements EmailRepository {
+
+    private Connection connection;
+    private Statement statement=null;
+    private ResultSet resultSet=null;
+    private ArrayList<Email> emails=new ArrayList<Email>();
```

Comment:
It is redundant to specify generic type on the right side, if generic is specified on the left side.